doc/reduce-traffic.md

```diff
@@ -35,3 +35,9 @@ blocks and transactions to fewer nodes.
 Reducing the maximum connected nodes to a minimum could be desirable if traffic
 limits are tiny. Keep in mind that bitcoin's trustless model works best if you are
 connected to a handful of nodes.
+
+## 4. Turn off transaction relay (`-blocksonly`)
+
+Forwarding transactions to peers increases the P2P traffic. To only sync blocks
+with other peers, you can disable transaction relay. Be reminded that this
+setting could hurt your privacy if used while a wallet is loaded.
```

Comment:
It could elaborate a bit on how it hurts privacy for those that aren't aware of the issue?

Reply:
We might want to also add that it greatly slows block propagation speed. (just so we don't get miners thinking this will optimize their nodes. :) )